prow/config/config.go

```diff
@@ -180,6 +180,13 @@ type ProwConfig struct {
 	// ManagedWebhooks contains information about all github repositories and organizations which are using
 	// non-global Hmac token.
 	ManagedWebhooks ManagedWebhooks `json:"managed_webhooks,omitempty"`
+
+	// ProwJobDefaultEntries holds a list of defaults for specific values
+	// Each entry in the slice specifies Repo and CLuster regexp filter fields to
+	// match against the jobs and a corresponding ProwJobDefault . All entries that
+	// match a job are used. Later matching entries override the fields of earlier
+	// matching entires.
+	ProwJobDefaultEntries []*ProwJobDefaultEntry `json:"prowjob_default_entries,omitempty"`
```

Comment:
Similiarly, why is this not simply called `TenantIDEntry`? We have a huge config and having an an option somewhere that is called `DefaultEntry` is pretty unclear about what it is.

Do we plan to extend this with other config options? If so, maybe re-use/extend the existing Preset mechanism?

Reply:
We want to leave it open to extend with other options as being able to set prowjob fields based on cluster or org/repo is useful. We can't use defaultDecorationConfig as it is not useable by all prowjobs, and we need tenantID (as well as other future fields) to be applied to all prowjobs.